apps/autoscheduler/CMakeLists.txt

```diff
@@ -153,5 +153,14 @@ endif()
 
 add_executable(test_perfect_hash_map test_perfect_hash_map.cpp)
 
+add_executable(test_function_dag test_function_dag.cpp FunctionDAG.h FunctionDAG.cpp ASLog.h ASLog.cpp)
+target_include_directories(
+  test_function_dag
+  PRIVATE "${HALIDE_INCLUDE_DIR}" "${HALIDE_TOOLS_DIR}")
+target_link_libraries(test_function_dag PRIVATE Halide)
+#if (NOT WIN32)
```

Comment:
Either remove entirely or add comment about why it should stay commented out

Reply:
I deleted the code.